### PR TITLE
Adding terms for v2

### PR DESCRIPTION
**Pull request by n3wscott**
Signed-off-by: Scott Nichols <snichols@vmware.com>

### CODE REVIEW

File: docs/SDK_v2.md [hunk 1]
```diff
@@ -1,3 +1,57 @@
...
+- [Event](https://github.com/cloudevents/spec/blob/master/spec.md#event), an
+  Event is the conical form of the attributes and payload of the occurrence.
+- [Message](https://github.com/cloudevents/spec/blob/master/spec.md#message), a
+  Message is the encoded form of an Event for a given encoding and protocol.
+- [Protocol](https://github.com/cloudevents/spec/blob/master/spec.md#protocol),
+  a Protocol is the over-the-wire format that Messages are sent.
+- [Protocol Binding](https://github.com/cloudevents/spec/blob/master/spec.md#protocol-binding),
+  a Protocol Binding defines how Events are mapped into Messages for the given
+  Protocol.
+- Client, a client contains the logic of interacting with a Protocol to enable
```
Comment (slinkydeveloper) [inline docs/SDK_v2.md]: Are u missing a link here?

File: docs/SDK_v2.md [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+- [Consumer](https://github.com/cloudevents/spec/blob/master/spec.md#consumer),
+  a "consumer" receives the event and acts upon it. It uses the context and data
+  to execute some logic, which might lead to the occurrence of new events.
+- [Intermediary](https://github.com/cloudevents/spec/blob/master/spec.md#intermediary),
+  An "intermediary" receives a message containing an event for the purpose of
+  forwarding it to the next receiver, which might be another intermediary or a
+  Consumer. A typical task for an intermediary is to route the event to
+  receivers based on the information in the Context.
+
+## Interaction Models
```
Comment (slinkydeveloper) [inline docs/SDK_v2.md]: Not sure what that means

File: docs/SDK_v2.md (hunk 1 above)
Comment (n3wscott) [reply docs/SDK_v2.md]: The spec does not define a client.

**Comment by slinkydeveloper** [comment]
LGTM

**Pull request merged**